cmd/geth/usage.go

```diff
@@ -211,7 +211,7 @@ var AppHelpFlagGroups = []flags.FlagGroup{
 		Flags: metricsFlags,
 	},
 	{
-		Name:  "WHISPER (EXPERIMENTAL)",
+		Name:  "WHISPER (deprecated)",
```

Comment:
Maybe move this under the `show-deprecated-flags` command since the functionality is deprecated.

Reply:
^ this?